src/features/game/dialogue/GameDialogueManager.ts

```diff
@@ -79,6 +80,7 @@ export default class DialogueManager {
 
     // Disable interactions while processing actions
     GameGlobalAPI.getInstance().enableSprite(this.getDialogueRenderer().getDialogueBox(), false);
+    this.getInputManager().enableKeyboardInput(false);
```

Comment:
I believe that the enabling and disabling input in GameActionExecutor is because we currently disable any keyboard inputs while attempting the quiz to prevent its interference with the normal proceeding of dialogue.